### DISCUSSION TITLE
Is it possible to hide a tray icon of a sandboxed program

### DISCUSSION THREAD

**Discussion by lroncrown** [sandboxie-plus/Sandboxie#1573] [Q&A Feedback]
Hello,
i am using Sandboxie Plus to have 2 instances of steam open, so I can play a game on a second account but still have my friends list open from my main account. Now I only need the Steam tray icon of my my steam.exe but if I try to hide the sandboxed steams icon it hides both. Is it possible to unlink both of these or is there another work around for my problem.

Thank you :)

**Comment by DavidXanatos** [comment]
You can't unkiln them as windows uses the file path to link them.
ergo, you need to make a copy of steam.exe in an otehr location, or well amy be even in the same, just call it steam1.exe and for windows these should be two independant apps
